assets/images/game-ci-brand-logo-wordmark.svg

```diff
@@ -1,2 +1,119 @@
-<svg id="Logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 691.59 382.06">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="1024mm"
+   height="1024mm"
```

Comment:
We should probably minimise this SVG. Things like inkscape and print size are not necessary for the web - and this svg specifically will increase the initial page load time, as it's on all pages.